docs/remotely-debuggable-browsers.md

```diff
@@ -1,5 +1,29 @@
 ## Remotely debuggable browsers
 
+#### Table Of Contents
+
+[Firefox](#firefox)
+  * [MacOS](#macos)
+    * [Firefox](#firefox-release)
+    * [Firefox Beta](#firefox-beta)
+    * [Firefox Developer Edition](#firefox-developer-edition-aurora)
+    * [Firefox Nightly](#firefox-nightly)
```

Comment:
I agree it is repetitive.  How about I leave two commands for for Firefox and Nightly to make it less repetitive and have notes about DE and Beta.  I think you said once how you like being able to copy the command directly from the doc is nice and I would definitely miss those two versions.  Plus it keeps it similar to the Chrome listing.